packages/pluggableWidgets/html-node-web/src/HTMLNode.xml

```diff
@@ -0,0 +1,325 @@
+                                <enumerationValue key="onPointerOver">onPointerOver</enumerationValue>
+                                <enumerationValue key="onPointerOverCapture">onPointerOverCapture</enumerationValue>
+                                <enumerationValue key="onPointerOut">onPointerOut</enumerationValue>
+                                <enumerationValue key="onPointerOutCapture">onPointerOutCapture</enumerationValue>
+                                <enumerationValue key="onGotPointerCapture">onGotPointerCapture</enumerationValue>
+                                <enumerationValue key="onGotPointerCaptureCapture">onGotPointerCaptureCapture</enumerationValue>
+                                <enumerationValue key="onLostPointerCapture">onLostPointerCapture</enumerationValue>
+                                <enumerationValue key="onLostPointerCaptureCapture">onLostPointerCaptureCapture</enumerationValue>
+                                <enumerationValue key="onScroll">onScroll</enumerationValue>
+                                <enumerationValue key="onScrollCapture">onScrollCapture</enumerationValue>
+                                <enumerationValue key="onWheel">onWheel</enumerationValue>
+                                <enumerationValue key="onWheelCapture">onWheelCapture</enumerationValue>
+                                <enumerationValue key="onAnimationStart">onAnimationStart</enumerationValue>
+                                <enumerationValue key="onAnimationStartCapture">onAnimationStartCapture</enumerationValue>
+                                <enumerationValue key="onAnimationEnd">onAnimationEnd</enumerationValue>
+                                <enumerationValue key="onAnimationEndCapture">onAnimationEndCapture</enumerationValue>
+                                <enumerationValue key="onAnimationIteration">onAnimationIteration</enumerationValue>
+                                <enumerationValue key="onAnimationIterationCapture">onAnimationIterationCapture</enumerationValue>
+                                <enumerationValue key="onTransitionEnd">onTransitionEnd</enumerationValue>
+                                <enumerationValue key="onTransitionEndCapture">onTransitionEndCapture</enumerationValue>
+                                <!-- END REACT EVENTS -->
+                            </enumerationValues>
+                        </property>
+                        <property key="eventAction" type="action" required="false">
+                            <caption>Action</caption>
+                            <description />
+                        </property>
+                        <property key="eventActionRepeat" type="action" required="false" dataSource="../tagContentRepeatDataSource">
+                            <caption>Action</caption>
+                            <description />
+                        </property>
```

Comment:
We need to add short description on what makes `eventActionRepeat` different from `eventAction`